Warn & Disallow Creating Role with Existing Name (#132218)
elastic/kibana: docs/api/role-management/put.asciidoc

  To grant access to all spaces, set to `["*"]`, or omit the value.
=====

[[role-management-api-put-query-params]]
==== Query parameters

`createOnly`::

>   (Optional, boolean) When `true`, will disallow operation on an existing role.

Reviewer: ```suggestion
  (Optional, boolean) When `true`, will prevent overwriting the role if it already exists.
```
